tests/integration/webserver/upgrade/src/main/resources/certGen.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+
+set -e
+
+FOLDER=./
+ALIAS='1'
+PASSWORD=${1:-'password'}
+
+function clean(){
+    FILE=$1
+    echo "=============================> Cleaning $FILE"
+    rm -f ${FILE}
+}
+
+function printKeystore(){
+    KEYSTORE=$1
+    echo "=============================> Print ${KEYSTORE}"
+    keytool -list -keystore ${FOLDER}/${KEYSTORE} -storepass ${PASSWORD} -v
```

Comment:
Why the switch from `openssl` to `keytool` ? Instructions for openssl are ubiquitous.
Also, we should be consistent and have instructions or a script in each test where we have pkcs#12  files checked-in.

I would personally vote for a `README.md` in each test with dedicated instructions.

Reply:
I missed the keytool usage. Agree with @romain-grecourt  on this one. I did not want to have a script, as anything executable is a pain to maintain. Instructions should be enough.

Reply:
@romain-grecourt the reason to have it in keytool is because it exists in JDK, and JDK is needed in Helidon too. Openssl is not installed in Windows as far as I know.

@tomas-langer what is the difference in maintenance a script or a readme?. I guess you had some bad experiences in the past but I am curious to know what could it be.

Reply:
`openssl` is more widely used, more versatile, and is applicable to non Java use-cases.
E.g. If we need to document end to end SSL encryption for users, we should use openssl. In this case having all command line instructions (public docs, and internal dev readmes) based on `openssl` is more consistent.

---

We don't need cross platform instructions (E.g. we use bash everywhere) ; `openssl` is not a build requirement, it is easy to install if needed.

Reply:
Okay, just note that we cannot create truststores in openssl (or that is my understanding), so we will need to do this:
```
        CertificateFactory cf = CertificateFactory.getInstance("X.509");
        X509Certificate cert = (X509Certificate) cf.generateCertificate(Resource.create("certificate.pem").stream());
        Tls clientTls = Tls.builder()
                .trust(trust -> trust.addCert(cert))
                .build();
```

Instead of this:
```
        Tls clientTls = Tls.builder()
                .trust(trust -> trust
                        .keystore(store -> store
                                .passphrase("password")
                                .trustStore(true)
                                .keystore(Resource.create("client.p12"))))
                .build();
```

Reply:
It is done

Reply:
You don't need to use `.pem` ; see `certification_notes.md`.
Also, can you add a README.md for each test that has a checked-in keystores.

Reply:
I added the readmes.

But in the certificate_notes.md I don't see where it is written what the client should use. It cannot use the private key and the server keystore, because that is private information of the server.